testbed/pyproject.toml

```diff
@@ -63,6 +63,14 @@ flatpak_runtime = "org.gnome.Platform"
 flatpak_runtime_version = "42"
 flatpak_sdk = "org.gnome.Sdk"
 
+[tool.briefcase.app.testbed.freeBSD]
```

Comment:
This won't work until there's a freeBSD backend for Briefcase; and it wouldn't be needed anyway, because it exists to test *GTK*, not Linux.